bin/local-status-check

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+puts "Starting #{__FILE__} happening now."
```

Comment:
For consistency with the output at the end, perhaps this should read `"Started #{__FILE__} checks..."`